pkg/kubectl/cmd/edit.go

```diff
@@ -120,12 +120,18 @@ func RunEdit(f *cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 		return err
 	}
 
+	// NB: we use `f.Decoder(false)` to get a plain deserializer for
```

Comment:
I would suggest moving this comment to immediately before the f.Decoder(false) line.